code/mob/living/critter/spider.dm

```diff
@@ -359,6 +436,31 @@
 		// egg_path = /obj/item/reagent_containers/food/snacks/ingredient/egg/critter/cluwne
 		max_defensive_babies = 150
 
+		critter_attack(target)
+			if(ismob(target))
+				var/datum/targetable/critter/spider_bite/bite = src.abilityHolder.getAbility(/datum/targetable/critter/spider_bite/cluwne)
+				var/datum/targetable/critter/clownspider_trample/trample = src.abilityHolder.getAbility(/datum/targetable/critter/clownspider_trample/cluwne)
+				if (!trample.disabled && trample.cooldowncheck() && prob(20))
+					trample.handleCast(target)
+				else if(!bite.disabled && bite.cooldowncheck())
+					bite.handleCast(target)
+				else
+					src.set_a_intent(INTENT_HARM)
+					src.hand_attack(target)
+
+		critter_scavenge(target)
+			var/datum/targetable/critter/spider_drain/drain = src.abilityHolder.getAbility(/datum/targetable/critter/spider_drain/cluwne)
+			if(!drain.disabled && drain.cooldowncheck())
+				return !drain.handleCast(target)
+
+		can_critter_scavenge()
+			var/datum/targetable/critter/spider_drain/drain = src.abilityHolder.getAbility(/datum/targetable/critter/spider_drain/cluwne)
+			return (!drain.disabled && drain.cooldowncheck())
+
+		can_critter_attack()
+			var/datum/targetable/critter/clownspider_trample/trample = src.abilityHolder.getAbility(/datum/targetable/critter/clownspider_trample/cluwne)
+			return !trample.disabled
```

Comment:
can it not hand attack?

Reply:
it can, just not while trampling (basically clown flail)